setup.py

```diff
@@ -47,12 +48,23 @@ def _build_paneljs():
         fcntl.fcntl(sys.stdout, fcntl.F_SETFL, flags&~os.O_NONBLOCK)
 
 
+def _install_jupyter_server_extension():
+    # data_files are not copied for editable installs returning
+    files = setup_args['data_files']
+    for dst, (src,) in files:
+        dst = os.path.join(sys.prefix, dst)
+        os.makedirs(dst, exist_ok=True)
+        shutil.copy2(src, dst)
+        print(f"Copied {src} to {dst}")
```

Comment:
A unit test for this could be:

``` python
def test_jupyter_config():
        jp_files = Path(sys.prefix) / 'etc' / 'jupyter').rglob('panel-client-jupyter.json')
        assert len(list(jp_files)) == 2
```